xrpl/asyncio/transaction/main.py

```diff
@@ -25,7 +25,7 @@
 _LEDGER_OFFSET: Final[int] = 20
 
 # TODO: make this dynamic based on the current ledger fee
-_ACCOUNT_DELETE_FEE: Final[int] = int(xrp_to_drops(5))
+_ACCOUNT_DELETE_FEE: Final[int] = int(xrp_to_drops(2))
```

Comment:
should we just get rid of this now? this lib should probably always ask the ledger for the current fee for sugar methods. you'd use a transaction yourself if you want to specify a custom fee

Reply:
I agree - but as of right now, the autofill method does not require a client, so there needs to be some default value for that case. IMO the autofill logic needs a lot of re-working, based on our findings from xrpl.js. But that is way beyond the scope of this PR/ticket.